src/SayMore/Transcription/UI/SegmentingAndRecording/OralAnnotationRecorderDlgBase.cs

```diff
@@ -1869,8 +1878,12 @@ private Rectangle PlayAnnotationButtonRectangle
 
 				// SP-1000: Reduce minimum segment size
 				// Reducing the minimum segment size below 850 ms requires repositioning the Play Annotation button
+				if (rc.Width < 45)
+					return new Rectangle(rc.Left + 3, rc.Bottom - 40 - _playButtonSize.Height,
```

Comment:
I believe the magic numbers "45", "80", and "40" based on the size of the two buttons? If so, could we use those sizes  instead? Also, to ensure the two buttons stay aligned (i.e, that consistent margins are used), I think it would probably be good to create a method that returns a rectangle, given a bounding rectangle (rc), an offset from bottom, and a button size. It could even include the logic (from higher up) to return an empty rectangle if (rc.IsEmpty || xxxButtonSize.Width + 6 > rc.Width). We could either call it only if the width is less than the width of the two buttons plus the minimum required margin or we could have it also take a "normal" left offset to be used when the buttons fit side by side, in which case it would also nicely handle the final calculation for the non-squeezed case.